composer.json

```diff
@@ -21,14 +21,14 @@
         "ext-curl":"*",
         "phpcr/phpcr": "~2.1.2",
         "phpcr/phpcr-utils": "^1.2.8",
-        "jackalope/jackalope": "~1.2.5"
+        "jackalope/jackalope": "~1.3"
     },
     "provide": {
-        "jackalope/jackalope-transport": "1.1.0"
+        "jackalope/jackalope-transport": "dev-master"
     },
     "require-dev": {
         "psr/log": "~1.0",
-        "phpcr/phpcr-api-tests": "2.1.11",
+        "phpcr/phpcr-api-tests": "dev-master",
```

Comment:
i merged https://github.com/phpcr/phpcr-api-tests/pull/175 and tagged 2.1.12 - please change dev-master to 2.1.12 and remove phpcr-api-tests from the repositories redirection below.